pkg/demoinfocs/common/equipment.go

```diff
@@ -118,6 +118,25 @@ const (
 	EqWrench                   EquipmentType = 419
 	EqSnowball                 EquipmentType = 420
 	EqBumpMine                 EquipmentType = 421
+	EqKnifeBayonet             EquipmentType = 422
+	EqKnifeCSS                 EquipmentType = 423
+	EqKnifeFlip                EquipmentType = 424
+	EqKnifeGut                 EquipmentType = 425
+	EqKnifeKarambit            EquipmentType = 426
+	EqKnifeM9Bayonet           EquipmentType = 427
+	EqKnifeTactical            EquipmentType = 428
+	EqKnifeFalchion            EquipmentType = 429
+	EqKnifeSurvivalBowie       EquipmentType = 430
+	EqKnifeButterfly           EquipmentType = 431
+	EqKnifePush                EquipmentType = 432
+	EqKnifeCord                EquipmentType = 433
+	EqKnifeCanis               EquipmentType = 434
+	EqKnifeUrsus               EquipmentType = 435
+	EqKnifeGypsyJackknife      EquipmentType = 436
+	EqKnifeOutdoor             EquipmentType = 437
+	EqKnifeStiletto            EquipmentType = 438
+	EqKnifeWidowmaker          EquipmentType = 439
+	EqKnifeSkeleton            EquipmentType = 440
```

Comment:
the `RawEquipment` part looks good now - but this part will need to be removed as it's not backwards compatible as mentioned in the other thread.

So all we should do for now is expose `RawEquipment`.

Maybe we can have a separate type `KnifeType` and constants `KnifeTypeXYZ` with a map `KnifeTypes map[string]KnifeType` which includes all known knife types to still provide some kind of mapping.

The good thing about this is that if a new knife comes out the users can just add it to the map themselves,. as the map will be mutable.

I think this would be the best middleground